pkg/v1/tkg/client/cluster.go

```diff
@@ -539,7 +539,7 @@ func (c *TkgClient) ConfigureAndValidateWorkloadClusterConfiguration(options *Cr
 		// NOTE: Not blocking the workload cluster deployment if the pinniped information is not available on management cluster
 		pinnipedIssuerURL, pinnipedIssuerCAData, err := clusterClient.GetPinnipedIssuerURLAndCA()
 		if err != nil {
-			log.Warningf("Warning: Pinniped configuration not found. Skipping pinniped configuration in workload cluster. Please refer to the documentation to check if you can configure pinniped on workload cluster manually")
+			log.Warningf("Warning: Pinniped configuration not found; Authentication via Pinniped will not be setup in this cluster. If you wish to setup Pinniped after the cluster is created, please refer to the documentation.")
```

Comment:
just a tiny language nit, feel free to merge after correcting:
```suggestion
			log.Warningf("Warning: Pinniped configuration not found; Authentication via Pinniped will not be set up in this cluster. If you wish to set up Pinniped after the cluster is created, please refer to the documentation.")
```